components/automate-ui/src/app/entities/service-groups/service-groups.reducer.ts

```diff
@@ -52,6 +57,17 @@ export function serviceGroupEntityReducer(
       return set('filters', filters)(state);
     }
 
+    case ServiceGroupsActionTypes.GET_SERVICE_GROUPS_COUNTS:
+    return set('status', EntityStatus.loading, state);
+
+    case ServiceGroupsActionTypes.GET_SERVICE_GROUPS_COUNTS_SUCCESS:
+    return pipe(
+      set('status', EntityStatus.loadingSuccess),
+      set('serviceGroupHealthCounts', action.payload))(state);
+
+    case ServiceGroupsActionTypes.GET_SERVICE_GROUPS_COUNTS_FAILURE:
+    return set('status', EntityStatus.loadingFailure, state);
```

Comment:
Since we are passing an HttpErrorResponse, lets at least store it into the redux state like:
```suggestion
      return pipe(
        set('status', EntityStatus.loadingFailure),
        set('errorResp', action.payload))(state);
```